source/crud/bulk-write.md

```diff
@@ -638,15 +638,10 @@ write concern containing the following message:
 
 The server reports a `maxBsonObjectSize` in its `hello` response. This value defines the maximum size for documents that
 are inserted into the database. Documents that are sent to the server but are not intended to be inserted into the
-database (e.g. command documents) have a size limit of `maxBsonObjectSize + 16KiB`. When an acknowledged write concern
-is used, drivers MUST NOT perform any checks related to these size limits and MUST rely on the server to raise an error
-if a limit is exceeded. However, when an unacknowledged write concern is used, drivers MUST raise an error if one of the
-following limits is exceeded:
-
-- The size of a document to be inserted MUST NOT exceed `maxBsonObjectSize`. This applies to the `document` field of an
-    `InsertOneModel` and the `replacement` field of a `ReplaceOneModel`.
-- The size of an entry in the `ops` array MUST NOT exceed `maxBsonObjectSize + 16KiB`.
-- The size of the `bulkWrite` command document MUST NOT exceed `maxBsonObjectSize + 16KiB`.
+database (e.g. command documents) have a size limit of `maxBsonObjectSize + 16KiB`. Clients MUST NOT validate the size
```

Comment:
Consider just removing the entire section, as it seems unnecessary to mention the size limit only to say drivers must not include any code that enforces it.

Reply:
I simplified the section, but I think the section should stay for the specification to be explicit regarding the requirement to rely on the server validating the document size.

Reply:
Removed the section in 125d1921cda0acefb96f8160b42499037515e674.